source/partials/homepage/_hero.slim

```diff
@@ -1,18 +1,26 @@
-.u-smallMargin
-h1.u-visuallyHidden We are Subvisual
-.ContentFormatter.ContentFormatter--centerHorizontally.u-onlySmall
-  img.RatioEnforcer(
-    src="#{image_path('homepage/hero-mobile.png')}"
-    srcset="#{image_path('homepage/hero-mobile.png')} 320w, #{image_path('homepage/hero-mobile@2x.png')} 640w"
-    sizes="100vw"
-    alt="Another galaxy"
-  )
-.u-exceptSmall
-  object#HomepageHero.HoverAnimation data="/images/homepage/hero.svg" type="image/svg+xml"
-.u-smallMargin
-.GridCell.GridCell--alignCenter.u-defaultThenDoubleLargeMargin
-  .GridCell-content
-    .ContentFormatter.ContentFormatter--centerHorizontally
-      .u-smallThenDefaultMargin
-      p.Text.Text--dark
-        | We handcraft thoughtful digital experiences that empower individuals, institutions and communities
+#HomepageHero.Panel.Panel--homepage
+  .Panel-content
+    .HomepageHero
+      .HomepageHero-content
+        .HomepageHero-copy
+          h1.HomepageHero-heading.u-xSmallThenSmallMargin
+            ' We
+            span.HomepageHero-headingHighlight> design
+            ' and
+            span.HomepageHero-headingHighlight> develop
+            ' thoughtful digital
+            span.HomepageHero-headingHighlight products
+          p.HomepageHero-text.u-defaultMargin
+            | We craft products with a focus on great User Experience and high-quality code
```

Comment:
About this copy, what about going more in the direction of what the client can get from us and not what we can give to the client, if that makes sense. /cc @johnny1337